pkg/router/template/router.go

```diff
@@ -512,12 +512,27 @@ func (r *templateRouter) writeConfig() error {
 		r.state[k] = cfg
 	}
 
+	data, err := ioutil.ReadFile("/proc/sys/net/ipv4/neigh/default/gc_thresh3")
+	if err != nil {
+		glog.Warningf("Error reading ARP neighbour information: %s", err)
```

Comment:
I only want to log this once ever.  Can we store something so we don't log for this twice?

Reply:
A lot of this is data that really isn't part of the template router.  It should really be separate and hidden from the details of the template router.  At a minimum you should treat it as a thing run side by side that generates warning.s

Reply:
since here is where the endpoints are added to the template, i think it's a proper place to do this check @knobunc what do you think?

Reply:
I'm speaking about code structure.  Arp checking has nothing to do with templatizing content.  Move this into a package that is distinct, and define the minimum abstraction necessary to tie this code to the template router.  This code is too Linux specific, too tied to one router impl, and would be likely required in future routers / ingress controllers.  Keep it separate.

Reply:
And possibly move it up to a higher level / turn it into a metric.  Can we not check the "fullness" of the arp cache from a given node?  If so, make this a metric for prometheus